app/models/ddr/batch/batch_object.rb

```diff
@@ -245,6 +256,11 @@ def clear_attributes(repo_object, attribute)
         return repo_object
       end
 
+      def add_role(repo_object, role)
+        repo_object.roles.grant([ scope: role.role_scope, type: role.role_type, agent: role.agent ])
```

Comment:
OK, I see this actually works -- it's Ruby :) -- but the brackets are redundant and IMO look weird. Do you mind changing it?